organisaatio-ui/src/OrganisaatioApp.tsx

```diff
@@ -60,35 +60,55 @@ const OrganisaatioApp: React.FC = () => {
     const [{ data: organisaatioTyypit, loading: organisaatioTyypitLoading, error: organisaatioTyypitError }] = useAxios<
         Koodi[]
     >(`${urlPrefix}/koodisto/ORGANISAATIOTYYPPI/koodi`);
+
+    const [{ data: ryhmanTilat, loading: ryhmanTilatLoading, error: ryhmanTilatError }] = useAxios<Koodi[]>(
+        `${urlPrefix}/koodisto/RYHMANTILA/koodi`
+    );
+
     if (
         languageLoading ||
         lokalisointiLoading ||
         kunnatLoading ||
         ryhmaTyypitLoading ||
         kayttoRyhmatLoading ||
-        organisaatioTyypitLoading
+        organisaatioTyypitLoading ||
+        ryhmanTilatLoading
```

Comment:
not really mandatory but this could be written like:
```
if ( loading() ) {
    return spinner();
}
```

This would make it easier to:
- add new resources
- unit test spinner view

Reply:
i suggest making a separate ticket for this.